ext-src/services/IqRequestService.ts

```diff
@@ -58,20 +59,28 @@ export class IqRequestService implements RequestService {
           url: `${this.url}/api/v2/applications?publicId=${applicationPublicId}`,
           headers: RequestHelpers.getUserAgentHeader(),
           auth: { user: this.user, pass: this.password }
-        },
-        (err: any, response: any, body: any) => {
-          if (err) {            
-            reject(`Unable to retrieve Application ID: ${err}`);
+        })
+        .on('response', (res) => {
```

Comment:
I switched to using the `.on('message', (handler) => {})` request for the `request` lib, as this is I think how it's preferred to use it. It certainly makes the flow read a bit easier, and leans on the framework. I mostly did this trying to figure out how to throw an error, etc.... but I liked the change enough to keep it.